source/transfer_list.rst

```diff
@@ -218,24 +213,19 @@ The intent of the void entry type is for information to be removed from the list
      - 0x0
      - The tag_id field must be set to **0**.
 
-   * - hdr_size
-     - 0x4
-     - 0x4
-     - The size of this entry header in bytes.
-
    * - data_size
+     - 0x3
      - 0x4
-     - 0x8
-     - The size of the data content in bytes.
+     - The size of the void space in bytes.
 
-   * - reserved
-     - 0x4
-     - 0xc
-     - Reserved, must be zero.
+   * - hdr_size
+     - 0x1
+     - 0x7
+     - The size of this entry header in bytes must be set to **8**.
 
    * - void_data
      - data_size
-     - hdr_size
+     - 0x8
```

Comment:
For this and all the other entries, I think the data offset should be defined as hdr_size. For the doc, the offset will always be 8. Since we have a hdr_size field, we implicitly assume that that offset can change in the future.

Reply:
Well, it means the same, I thought the table would be easier to read with absolute numbers. But I can change it back if you prefer.

Reply:
My main concern is the duplication, which can be resolved with the rst substitution feature (see other comment)

Reply:
Correct me if I'm wrong, from the above comment we're in agreement that we use literal 8 for now.
If that's correct, then this "conversation" can be "closed".

Reply:
After looking at this again I think I see the value in writing "hdr_size", which might help clarify that readers actually need to do the math with the hdr_size field from the header to find the data portion rather than just hardcode a number like 8. So I reverted to not changing this in the latest version of the PR.